internal/forms/forms.go

```diff
@@ -750,6 +863,18 @@ func (m *Manager) fillFormTemplate(absPathTemplate string, formDestURL string, p
 		l = strings.ReplaceAll(l, "{UDTG}", udtg)
 		l = strings.ReplaceAll(l, "{Time}", nowTime)
 		l = strings.ReplaceAll(l, "{UTime}", nowTimeUTC)
+		l = strings.ReplaceAll(l, "{GPS}", gpsFmt(degreeMinute, nowPos))
+		l = strings.ReplaceAll(l, "{GPS_DECIMAL}", gpsFmt(decimal, nowPos))
+		l = strings.ReplaceAll(l, "{GPS_SIGNED_DECIMAL}", gpsFmt(signedDecimal, nowPos))
+		// Lots of undocumented tags found in the Winlink check in form.
+		// Note also various ways of capitalizing. Perhaps best to do case insenstive string replacements....
+		l = strings.ReplaceAll(l, "{Latitude}", fmt.Sprintf("%.4f", nowPos.Lat))
+		l = strings.ReplaceAll(l, "{latitude}", fmt.Sprintf("%.4f", nowPos.Lat))
+		l = strings.ReplaceAll(l, "{Longitude}", fmt.Sprintf("%.4f", nowPos.Lon))
+		l = strings.ReplaceAll(l, "{longitude}", fmt.Sprintf("%.4f", nowPos.Lon))
+		l = strings.ReplaceAll(l, "{GridSquare}", posToGridSquare(nowPos))
+		l = strings.ReplaceAll(l, "{GPSValid}", fmt.Sprintf("%s ", validPos))
+		// l = strings.ReplaceAll(l, "{InternetAvailable}", "TO DO")
```

Comment:
TODO?

Reply:
I went through the forms and found a bunch of undocumented insertion tags. One if them was `InternetAvailable` which presumably somehow determines if Internet connectivity is currently in place. I didn't know how to determine that, and it wasn't really relevant to getting the GPSd stuff working but I wanted to document that there is this (and perhaps other) insertion tags that still need to be implemented. I will remove this line and create an Issue instead.

Reply:
#338